tests/test_lab_config.py

```diff
@@ -16,3 +20,93 @@ def test_ilab_config_show(cli_runner: CliRunner) -> None:
     assert parsed
 
     assert configuration.Config(**parsed)
+
+
+def test_ilab_config_init_with_env_var_config(
+    cli_runner: CliRunner, tmp_path: pathlib.Path
+) -> None:
+    # create a dummy config file
+    cfg = configuration.get_default_config()
+    assert cfg.general.log_level == "INFO"
```

Comment:
nit: There are `logging` module symbols for all levels that you could use here instead of strings.